orange/src/main/java/com/hartwig/hmftools/orange/report/tables/SignatureAllocationTable.java

```diff
@@ -72,10 +75,7 @@ else if(sig2.equalsIgnoreCase(MISALLOC_SIGNATURE))
                 return sig1.compareTo(sig2);
             }
 
-            int sigNumber1 = Integer.parseInt(sig1.substring(3));
-            int sigNumber2 = Integer.parseInt(sig2.substring(3));
-
-            return Integer.compare(sigNumber1, sigNumber2);
+            return Double.compare(allocation1.allocation(), allocation2.allocation()) * -1;
```

Comment:
It is more typical to use `Double.compare(allocation2, allocation1)` if you want to order descending.